util.go

```diff
@@ -50,11 +57,21 @@ func writeLines(name string, lines []string, truncate bool) error {
 		if err != nil {
 			return fmt.Errorf("cannot open file for appending: %w", err)
 		}
-		defer file.Close()
+
+		// When it is not possible to close file, then return error.
+		// This is very important in situation, when system call close()
+		// returns EIO. It means that previously-uncommitted write(2)
+		// encountered an input/output error, and we lost data.
```

Comment:
Please consider this wordsmith-ed suggestion:

```
        // When it is not possible to close the file, then return the error
        // reported by the failed close() system call.
        // This is very important in the situation when the system call close()
        // returns EIO. This can be caused when the previously-uncommitted write(2)
        // encountered an input/output error, resulting in the possible loss of data.

```